### ISSUE TITLE
simple_form with info icons

### ISSUE THREAD

**Issue by maxhansson** [dulvie/emmy#186]
With differnt page-width the icon is not on the same row as the field.
An exampel is "import accounting_plan"

**Comment by qzio** [comment]
which icon? The "Help icon" (an "i" with a circle)?

**Comment by qzio** [comment]
Does this relate to Pull request #185 Issue #178 or Issue #149 in some way?

**Comment by maxhansson** [comment]
Yes it is the help icon.  Accounting_plan is one page were help text for fields is displayed.
